build.sbt

```diff
@@ -14,7 +14,7 @@ libraryDependencies ++= Seq(
   "org.scalikejdbc" %% "scalikejdbc" % "2.1.2",
   "org.scalatest" %% "scalatest" % "2.1.5" % "test",
   "com.googlecode.flyway" % "flyway-core" % "2.3",
-  "me.fornever" %% "platonus" % "0.2.1",
+  "me.fornever" %% "platonus" % "0.3",
```

Comment:
You know, we have the _rules_ these days. If'd there be two or more codingteam contributors (currently only me) - I would be ready to transfer it.